components/abtesting/toggle/src/index.js

```diff
@@ -0,0 +1,43 @@
+/* eslint eqeqeq: "off" */
+
+import React, {Component, PropTypes} from 'react'
+
+class AbTestToggle extends Component {
+  render () {
+    let {variation, children} = this.props
+
+    var filterFunc
+    if (variation) {
```

Comment:
I hate multiline ternaries but this one is so cute. Changed.
Comparaison is abstract cause it can be string or number (check propTypes)